inc/spbc-settings.php

```diff
@@ -2126,6 +2126,14 @@ function spbc_field_scanner__prepare_data__analysis_log(&$table){
                     $analysis_status = $row->analysis_status;
             }
 		    
+		    // Filter actions for approved files
+            if ($row->analysis_status === 'SAFE') {
```

Comment:
Do we need to compare this with APPROVED_BY_CT?

Reply:
NO, SAFE - not the same as APPROVED_BY_CT